core/idl/wallet/common/wallet.djinni

```diff
@@ -348,6 +357,20 @@ Account = interface +c {
     const EV_NEW_OP_WALLET_NAME: string = "EV_NEW_OP_WALLET_NAME";
     const EV_NEW_OP_ACCOUNT_INDEX: string = "EV_NEW_OP_ACCOUNT_INDEX";
     const EV_NEW_OP_UID: string = "EV_NEW_OP_UID";
```

Comment:
Is it possible to retrieve UID when operation is removed from DB when a reorg happen ??

Reply:
I will add that as soon as possible